### PR TITLE
feat(web): redesign reports/single card summary design

### PR DESCRIPTION
**Pull request by mbezard**
Capture d’écran 2023-04-18 à 12 22 53

### CODE REVIEW

File: packages/web-reporter-ui/src/components/Score.tsx [hunk 1]
```diff
@@ -1,18 +1,73 @@
...
+          className="stroke-theme-color"
+        />
+        <text
+          x={100}
+          y={100}
+          dominantBaseline="middle"
+          textAnchor="middle"
+          className="text-center text-4xl font-semibold fill-white"
+        >
+          {`${roundToDecimal(score, 1)}%`}
```
Comment (Almouro) [inline packages/web-reporter-ui/src/components/Score.tsx:57]: I know it's in the design, but i'd vote to not keep the `%`, Lighthouse doesn't have it, what do you think?

File: packages/reporter/src/reporting/reporting.ts [hunk 2]
```diff
@@ -56,3 +56,11 @@ export const getAverageFPSUsage = (measures: Measure[]) =>
 
 export const getAverageRAMUsage = (measures: Measure[]) =>
   average(measures.map((measure) => measure.ram));
+
+export const getAverageTotalHightCPUUsage = (highCpuProcesses: {
```
[the remark this answers is not in the document]
Comment (mbezard) [reply packages/reporter/src/reporting/reporting.ts:60]: I have one but "Hight" exists in english ... 😅

[the remark this answers is not in the document]
Comment (mbezard) [reply packages/reporter/src/reporting/reporting.ts:60]: Capture d’écran 2023-04-18 à 14 10 17

[the remark this answers is not in the document]
Comment (Almouro) [reply packages/reporter/src/reporting/reporting.ts]: Wow, my bad, didn't realize that was an actual word 😅

File: packages/flipper-plugin-android-performance-profiler/src/__tests__/__snapshots__/Plugin.ts.snap [hunk 3]
```diff
@@ -5,45 +5,22 @@ exports[`displays FPS data and scoring 1`] = `
 Stop Measuring
 Reset
 Results
-Score
 47
 Average Test Runtime
-Time taken to run the test.
-Can be helpful to measure Time To Interactive of your app, if the test is checking app start for instance.
-1000ms
+84 ms
```
[the remark this answers is not in the document]
Comment (mbezard) [reply packages/flipper-plugin-android-performance-profiler/src/__tests__/__snapshots__/Plugin.ts.snap:10]: 😱

File: packages/flipper-plugin-android-performance-profiler/src/__tests__/__snapshots__/Plugin.ts.snap [hunk 4]
```diff
@@ -5,45 +5,22 @@ exports[`displays FPS data and scoring 1`] = `
...
+ v 
 Average FPS
-Frame Per Second. Your app should display 60 Frames Per Second to give an impression of fluidity. This number should be close to 60, otherwise it will seem laggy. 
-See
- 
-this video
- 
-for more details
-60
+375.8 FPS
```
[the remark this answers is not in the document]
Comment (mbezard) [reply packages/flipper-plugin-android-performance-profiler/src/__tests__/__snapshots__/Plugin.ts.snap:13]: 😱
it's good now i think. Wrong copypasting ...

File: packages/web-reporter-ui/src/components/ReportSummary/ReportSummary.tsx [hunk 5]
```diff
@@ -189,8 +189,103 @@ export const ReportSummary = ({
...
-      <SimpleTable rows={table} />
-    </>
+    <div className="flex flex-row overflow-x-scroll px-32 pt-12">
+      {averagedResults
+        .map((result) => (
+          <ReportSummaryCard key={result.name} averagedResult={result} />
+        ))
+        .reduce((acc, curr, index) => {
+          if (index === 0) return [curr];
+          return [...acc, <div className="w-24" />, curr];
```
[the remark this answers is not in the document]
Comment (mbezard) [reply packages/web-reporter-ui/src/components/ReportSummary/ReportSummary.tsx]: I ve just remembered I am not in RN and I can use flex-gap !

[the remark this answers is not in the document]
Comment (Almouro) [reply packages/web-reporter-ui/src/components/ReportSummary/ReportSummary.tsx]: oh yeah!!

File: packages/web-reporter-ui/src/components/ReportSummary/ReportSummary.tsx [hunk 6]
```diff
@@ -189,8 +189,103 @@ export const ReportSummary = ({
...
+  <div className="flex flex-row w-full px-6 py-4 border rounded-lg border-gray-800">
+    <div className="text-neutral-300">{title}</div>
+
+    <div className="grow" />
+
+    <p className="text-neutral-300 whitespace-pre">{`${value} ${unit}`}</p>
+
+    <div className="w-2" />
+
+    <div className="text-neutral-300"> v </div>
```
[the remark this answers is not in the document]
Comment (mbezard) [reply packages/web-reporter-ui/src/components/ReportSummary/ReportSummary.tsx]: Yes I'll do it later in an other PR

**Review by Almouro — APPROVED** [review]

**Pull request merged**